modules/dynunet_tutorial.ipynb

```diff
@@ -616,30 +577,26 @@
     "        )\n",
     "\n",
     "        def _compute_loss(preds, label):\n",
-    "            labels = [label] + [\n",
-    "                interpolate(label, pred.shape[2:]) for pred in preds[1:]\n",
-    "            ]\n",
+    "            preds = torch.unbind(preds, dim=1)\n",
     "            return sum(\n",
-    "                0.5 ** i * self.loss_function(p, l)\n",
-    "                for i, (p, l) in enumerate(zip(preds, labels))\n",
+    "                [\n",
```

Comment:
Small point, but list comprehension is unnecessary here as `sum` can take a generator as argument. i.e., you could delete the `[` and `]` and outdent the rest.